### ISSUE TITLE
out of date static dep

### ISSUE THREAD

**Issue by majestrate** [oxen-io/lokinet#2206]
zlib 1.2.13 is no longer available on zlib.net

**Comment by majestrate** [comment]
original report from IRC attached

```
[23:06] <xutaxkamay> jeff: seems like https://zlib.net/zlib-1.2.13.tar.xz does not exists anymore
[23:07] <xutaxkamay> just tried to build for openwrt
```

**Comment by dr7ana** [comment]
closed, zlib bumped to 1.3 in #2204
